tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Kafka/KafkaHelper.cs

```diff
@@ -17,6 +17,7 @@ internal static class KafkaHelper
     {
         private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor(typeof(KafkaHelper));
         private static bool _headersInjectionEnabled = true;
+        private static string[] defaultProduceEdgeTags = new[] { "type:kafka", "direction:out" };
```

Comment:
Must be alphabetically sorted
```suggestion
        private static string[] defaultProduceEdgeTags = new[] { "direction:out", "type:kafka" };
```